libs/langchain/langchain/schema/runnable/passthrough.py

```diff
@@ -37,11 +37,17 @@ async def ainvoke(
         return await self._acall_with_config(aidentity, input, config)
 
     def transform(
-        self, input: Iterator[Input], config: RunnableConfig | None = None
+        self,
+        input: Iterator[Input],
+        config: RunnableConfig | None = None,
```

Comment:
nice had no idea that pipe was usable

```
>>> a: str | None = 'hello'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
>>> from __future__ import annotations
>>> a: str | None = 'hello'
```

Reply:
Lol this was by mistake, no idea this would work in versions of python either